src/Stripe.net/Services/PreviewService.cs

```diff
@@ -0,0 +1,38 @@
+namespace Stripe.Services
+{
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class PreviewService
+    {
+        private readonly StripeClient client;
+
+        internal PreviewService(StripeClient client)
+        {
+            this.client = client;
+        }
+
+        public Task<StripeResponse> Get(string path, RawRequestOptions requestOptions = null, CancellationToken cancellationToken = default)
+        {
+            return this.client.RawRequestAsync(HttpMethod.Get, path, null, BuildPreviewOptions(requestOptions), cancellationToken);
+        }
+
+        public Task<StripeResponse> Post(string path, string body, RawRequestOptions requestOptions = null, CancellationToken cancellationToken = default)
+        {
+            return this.client.RawRequestAsync(HttpMethod.Post, path, body, BuildPreviewOptions(requestOptions), cancellationToken);
+        }
+
+        public Task<StripeResponse> Delete(string path, RawRequestOptions requestOptions = null, CancellationToken cancellationToken = default)
+        {
+            return this.client.RawRequestAsync(HttpMethod.Delete, path, null, BuildPreviewOptions(requestOptions), cancellationToken);
+        }
+
+        private static RawRequestOptions BuildPreviewOptions(RawRequestOptions requestOptions)
+        {
+            RawRequestOptions cloned = requestOptions != null ? (RawRequestOptions)requestOptions.Clone() : new RawRequestOptions();
+            cloned.ApiMode = ApiMode.Preview;
```

Comment:
Oh, I guess we don't need to hardcode the preview version when [there's already logic ](https://github.com/stripe/stripe-dotnet/pull/2690/files#diff-ac904c400fdd4a5fb6b51487f1d8b4f8b5cc1076c67c44b590c9b0ce53a5ecc3R186-R187)to set the Stripe-Version header if the API mode is preview.